src/vsr/replica.zig

```diff
@@ -8072,15 +8072,15 @@ pub fn ReplicaType(
             assert(self.superblock.working.vsr_state.sync_op_max > 0);
             assert(self.grid_repair_tables.available() > 0);
 
-            const snapshot_from_commit = vsr.Snapshot.readable_at_commit;
             const sync_op_min = self.superblock.working.vsr_state.sync_op_min;
             const sync_op_max = self.superblock.working.vsr_state.sync_op_max;
             while (self.sync_tables.?.next(&self.state_machine.forest)) |table_info| {
                 assert(self.grid_repair_tables.available() > 0);
                 assert(table_info.label.event == .reserved);
 
-                if (table_info.snapshot_min >= snapshot_from_commit(sync_op_min) and
-                    table_info.snapshot_min <= snapshot_from_commit(sync_op_max))
+                //FIXME
+                if (table_info.snapshot_min.timestamp >= Snapshot.readable_at_commit(sync_op_min) and
+                    table_info.snapshot_min.timestamp <= Snapshot.readable_at_commit(sync_op_max))
```

Comment:
`superblock.working.vsr_state.sync_op_min` and `superblock.working.vsr_state.sync_op_max`